### DISCUSSION TITLE
Upload action 支持自定义上传

### DISCUSSION THREAD

**Discussion by crazyair** [ant-design/ant-design#45779] [Ideas]
背景
如果想要自定义上传，目前是使用`customRequest`，但是此方式使用成本比较高，希望增加一个低成本方式，如：
```tsx
import axios from 'axios'
<Upload action={(formData) => axios.post(url, formData)} />
```
如果 action promise 已经被占用，可以想想其他的方式，目的是希望有一个方法，参数是`formData`，返回值是`Promise`

**Comment by zombieJ** [comment]
阶段不关注的话，可以直接用 `beforeUpload` 来做上传。

**Reply by crazyair** [reply]
这个也排除，就想拦截 ajax，上传用的 onChange
